Source/control.cpp

```diff
@@ -712,11 +742,11 @@ void DrawSpellList(const Surface &out)
 
 		switch (spellListItem.type) {
 		case RSPLTYPE_SKILL:
-			DrawSpellCel(out, spellListItem.location, *pSpellCels, SPLICONLAST + 3);
+			PrintSBookSpellType(out, spellListItem.location, N_("Skill"), PAL16_YELLOW + 1);

```

Comment:
N_() is a dummy macro for marking static content, since this isn't static you should be using the actual translation macro _()

Reply:
Thank you. Worked like a charm!